library/common-library/README.md

```diff
@@ -0,0 +1,95 @@
+# Helm Common library
+
+The common library is a way to unify the UX through all the Helm charts that implement it.
+
+The tooling suite that New Relic is huge and growing and this allows to set things globally
+and locally for a single chart.
+
+## Documentation for chart writers
+
+If you are writing a chart that is going to use this library you can check the [developers guide](/library/common-library/DEVELOPERS.md) to see all
+the functions/templates that we have implemented, what they do and how to use them.
+
+## Values managed globally
+
+Here is a list of options that you can set globally and affect all the charts in the `nri-bundle` (and this chart) at the same time while being
```

Comment:
Should we mention that it is possible that not all options have effect?

Reply:
Solved in [#f8fb902](https://github.com/newrelic/helm-charts/pull/797/commits/f8fb902c3fd8820c8131b947e5ee7a10fde0f8f7)